### PR TITLE
backport: Merge bitcoin#21187, 21342,17350,19238,21210,21274,(partial)21053,20370,20437,21388

### PR DESCRIPTION
**Pull request by vijaydasmp**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,8 +20,7 @@ source software which enables the use of this currency.
 Pre-Built Binary
 ----------------
 
-For more information, as well as an immediately usable, binary version of
-the Dash Core software, see https://www.dash.org/downloads/.
+For more information read the original Dash whitepaper.
```
Comment (UdjinM6) [inline README.md:23]: 21210: missing a (dashified) part of it https://github.com/bitcoin/bitcoin/pull/21210/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R6-R10

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
one tiny issue, LGTM otherwise

File: README.md (hunk 1 above)
Comment (vijaydasmp) [reply README.md:27]: Thanks I have corrected it

**Review by UdjinM6 — DISMISSED** [review]
utACK

**Comment by vijaydasmp** [comment]
Hello @knst, @PastaPastaPasta , requesting review

File: README.md (hunk 1 above)
Comment (knst) [reply README.md:27]: I think it should be not in chapter "Pre-Build Binary" but in chapter "What is Dash"? 

Probably need to remove these lines:
```
Pre-Built Binary
----------------
```

btw, @UdjinM6 what's is an original Dash whitepaper?

Comment (UdjinM6) [reply README.md:27]: > I think it should be not in chapter "Pre-Build Binary" but in chapter "What is Dash"?
> 
> Probably need to remove these lines:
> 
> ```
> Pre-Built Binary
> ----------------
> ```

Good catch!

> btw, @UdjinM6 what's is an original Dash whitepaper?

https://github.com/dashpay/dash/wiki/Whitepaper

but it's quite outdated now... maybe we should mention/link to DIPs here instead

**Comment by knst** [comment]
> Hello @knst, @PastaPastaPasta , requesting review

@vijaydasmp see https://github.com/dashpay/dash/pull/5529#discussion_r1293809816

**Comment by vijaydasmp** [comment]
> > Hello @knst, @PastaPastaPasta , requesting review
> 
> @vijaydasmp see [#5529 (comment)](https://github.com/dashpay/dash/pull/5529#discussion_r1293809816)

will check

**Comment by vijaydasmp** [comment]
Hello @knst   , I have updated as per comments

**Review by knst — APPROVED** [review]
utACK

**Review by UdjinM6 — APPROVED** [review]
utACK

**Review by PastaPastaPasta — APPROVED** [review]
utACK for merging via merge commit

**Pull request merged**